### ISSUE TITLE
Remove Bad Professors data

### ISSUE THREAD

**Issue by AdamMcAdamson** [UTDNebula/nebula-api#128]
There are some instances where the [Profiles](https://profiles.utdallas.edu/) and [Coursebook](https://coursebook.utdallas.edu/) professor data did not properly merge.

Notable filters:
```javascript
{sections: {$eq: []}}
```
```javascript
{profile_uri: {$eq: null}}
```

(I will include an example shortly)

We need to:
- [ ] Isolate instances of an improper merge
- [ ] Devise a solution to the merging issues
- [ ] Merge the data accordingly

**Comment by hochladen** [comment]
I'll help look into this.

As far as what I've checked so far goes, it seems like the professors with missing sections are the more apparent issue. I've been able to find professors who have multiple sections according to Coursebook without sections listed in the database. This is likely indeed an issue in the coursebook scraper merging, and I'll look further into it.

As far as the missing `profile_uri`s go, however, I haven't been able to find any professors that have a missing profile_uri despite having a profile. So far, all of the professors without a profile_uri haven't have a profile to begin with. Perhaps some examples would help here.
